docs/using-bi-for-auth.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using Beyond Identity for Authentication
+sidebar_position: 2
+---
+In this guide we will show you how to create a tenant, an application, configure the application, and how to integrate with the Beyond Identity Embedded SDK
```

Comment:
"In this guide, we will show you how to create a tenant and and application, as well as how to configure that application in order to authorize and authenticate against Beyond Identity"